<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow_imgViewer</class>
 <widget class="QMainWindow" name="MainWindow_imgViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>969</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow_imgViewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView_NIR">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>638</width>
      <height>510</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_RGB">
    <property name="geometry">
     <rect>
      <x>658</x>
      <y>10</y>
      <width>302</width>
      <height>302</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_depth">
    <property name="geometry">
     <rect>
      <x>656</x>
      <y>330</y>
      <width>302</width>
      <height>227</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>550</y>
      <width>621</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_open">
       <property name="text">
        <string>Open Dir</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_previous">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;&lt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_markAsBad">
       <property name="text">
        <string>Mark as &quot;bad&quot;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_next">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>&gt;&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_imageName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>621</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Image name</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>969</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
